MDANSE/Src/IO/HDF5.py

```diff
@@ -38,29 +39,29 @@ def get_attributes(self):
         """
         return self._variable.attrs
 
-
 def find_numeric_variables(var_dict, group):
     """
-    Recursively retrieves all the numeric variables stored in an HDF5 file.
+    Retrieves the numeric variables stored in an HDF5 file.
 
-    :param var_dict: The dictionary into which the variables are saved.
+    :param var_dict: dict into which the variables are saved.
     :type var_dict: dict
 
     :param group: The file whose variables are to be retrieved.
     :type group: h5py.File or h5py.Group
     """
 
-    for var_key, var in list(group.items()):
+    for var_key, var in group.items():
+
         if isinstance(var, h5py.Group):
             find_numeric_variables(var_dict, var)
         else:
-            if var.parent.name == "/":
-                path = "/{}".format(var_key)
+            if var.parent.name == '/':
+                path = '/{}'.format(var_key)
             else:
-                path = "{}/{}".format(var.parent.name, var_key)
+                path = '{}/{}'.format(var.parent.name, var_key)
 
             # Non-numeric variables are not supported by the plotter
-            if not np.issubdtype(var.dtype, np.number):
+            if not numpy.issubdtype(var.dtype, numpy.number):
```

Comment:
In line 17 we did
`import numpy as np`
It means that the `numpy` module is visible in the workspace under the name `np`.
Please do not replace `np` in the code with `numpy`, since the name `numpy` will not be recognised.

Reply:
np now consistent throughout the script.